.gitignore

```diff
@@ -28,6 +28,9 @@ python-coverage/
 cover
 _attic
 
+# developer mode toggle
+kbase-extension/static/kbase/js/util/developerMode.js
```

Comment:
If this is going into .gitignore, aren't there a bunch of files that'll break?

Reply:
It just means you can set dev mode on locally and not have to worry about it getting added to a git commit by accident.